buda/entities/visit.proto

```diff
@@ -20,6 +20,12 @@ message Visit {
         string term = 5;
     }
 
+    message Experiment {
+        string id = 1;
+        string name = 2;
+        bool enabled = 3;
```

Comment:
Curious, is meant to designate the experiment is enabled, or the visitor is part of the `enabled` group (as opposed to the `control` group)?  If it is the second, having this a boolean instead of the name of the group might limit us from multivariate testing 🤔 .

Reply:
@mallen5311 really great question. yeah, right now the experiments on the new marketing site only support simple experiments (experiment and control groups), I think, which is why I modelled it like this. 

I could always make this a string `group` field instead and then we could default to passing either the value `experiment` or `control` from the tracking side. This can 'future proof' us to support multiple groups.

I do kind of like that we are constrained to simple A/B tests for now, which I think will reduce complexity in designing and interpreting experiments.

Reply:
Thanks for sharing that context, @michael-erasmus !  That makes sense to me 😄 .